lutris/util/wine/proton.py

```diff
@@ -9,7 +9,8 @@
 from lutris.util import system
 from lutris.util.steam.config import get_steamapps_dirs
 
-GE_PROTON_LATEST = _("GE-Proton (Latest)")
+GE_PROTON_LATEST = _("UMU-Proton (Latest)")
+PROTON_LATEST = _("GE-Proton (Latest)")
```

Comment:
This is confusing because the identifier contradicts the values. But unfortunately, if we don't want to break user's wine prefixes in the next release, the identifiers for the values can't be changed. 

Unless Lutris wants to accept breaking prefixes, I can change them though.